This patch moves ld.so.cache from /etc to /var/run. This is for devices
where /etc is JFFS2 or CRAMFS but /var is a ramdisk.

#
# Patch managed by http://www.mn-logistik.de/unsupported/pxa250/patcher
#

--- libc/sysdeps/generic/dl-cache.h~ldsocache-varrun
+++ libc/sysdeps/generic/dl-cache.h
@@ -29,7 +29,7 @@
 #endif
 
 #ifndef LD_SO_CACHE
-# define LD_SO_CACHE SYSCONFDIR "/ld.so.cache"
+# define LD_SO_CACHE "/var/run/ld.so.cache"
 #endif
 
 #ifndef add_system_dir
